entrypoint.sh

```
@@ -1,28 +1,57 @@
-        DATABASE_URL="postgresql://${DATABASE_USERNAME}:${DATABASE_PASSWORD}@${DATABASE_HOST}/${DATABASE_NAME}"
+# Exit immediately if a command fails
+set -e
+# Disable auto-exporting of variables
+set +a
+
+# If a CONFIG_PATH is set, resolve the environment overrides from the config file.
+# The overrides will be written into variables scopped to the current shell. This is
+# required in case one of the variables used in this entrypoint is overriden (e.g.,
+# DATABASE_URL, REDIS_URL, etc.)
+if [ -n "$CONFIG_PATH" ]; then
+    echo -e "\e[34m[Info] Resolving environment overrides from $CONFIG_PATH...\e[0m"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)